titus-server-master/src/main/java/com/netflix/titus/master/mesos/kubeapiserver/KubeApiServerIntegrator.java

```diff
@@ -248,7 +248,15 @@ public void killTask(String taskId) {
         killTaskCounter.increment();
         try {
             logger.info("deleting pod: {}", taskId);
-            kubeApiFacade.getCoreV1Api().deleteNamespacedPod(taskId, KUBERNETES_NAMESPACE, null, null, null, DELETE_GRACE_PERIOD_SECONDS, null, null);
+            kubeApiFacade.getCoreV1Api().deleteNamespacedPod(taskId,
+                    KUBERNETES_NAMESPACE,
+                    null,
+                    null,
+                    DELETE_GRACE_PERIOD_SECONDS,
+                    null,
+                    null,
+                    null
+            );
         } catch (JsonSyntaxException e) {
             // this is probably successful. the generated client has the wrong response type
```

Comment:
should verify if extra catch is still needed

Reply:
Still fails with the same exception:
```java
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException:
 Expected a string but was BEGIN_OBJECT at line 1 column 2451 path $.status
```